src/Maker.js

```diff
@@ -85,7 +87,8 @@ function delegateToServices(maker, services) {
 
 function mergeOptions(object, source) {
   return mergeWith(object, source, (objValue, srcValue, key) => {
-    if (Array.isArray(objValue) && key !== 'abi') return objValue.concat(srcValue);
+    if (Array.isArray(objValue) && key !== 'abi')
+      return uniq(objValue.concat(srcValue));
```

Comment:
This was adding a duplicate to the `additionalServices` array, so I ran `uniq` on it. I looked at other configs and I can't think of any other side effects from removing dupes from the array.

Reply:
I added line 90 because abi properties that are arrays were being concatenated, resulting in duplicates for properties like `inputs` and `outputs`